tests/test_flow/generate_test_compatability_cache.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from ..helpers import count_calls
+
+import bionic as bn
+
+
+def get_test_dir():
```

Comment:
Since `__file__` is basically a constant, could/should this just be a top-level constant rather than a function?

Reply:
Makes sense to me